release_notes/ocp-4-20-release-notes.adoc

```diff
@@ -478,11 +478,30 @@ With this release, you can install a cluster on {azure-short} using Intel-based
 
 For more information, see xref:../installing/installing_azure/ipi/installing-azure-customizations.adoc#installation-azure-confidential-vms_installing-azure-customizations[Enabling confidential VMs].
 
+[id="ocp-release-notes-installation-oci-offerings_{context}"]
+==== Additional Oracle Distributed Cloud and Oracle Edge Cloud infrastructure offerings available as Technology Preview
+
+With this release, you can install a Technology Preview cluster on new Oracle infrastructure offerings.
+
+The following new Oracle Distributed Cloud infrastructure types are now available:
+
+* US Government Cloud
+* UK Government Cloud
```

Comment:
@skopacz1 Oracle has renamed this service; now it is called "Oracle UK Sovereign Cloud"